Modules/data.py

```diff
@@ -12,19 +12,24 @@
 aban = 550114698297737216
 shban = 416786636118949918
 
-verified = 346604542550605835
+verified = 762535941579341838

```

Comment:
Correct Id should be 346604542550605835.

Reply:
Made some changes just now, let me know if anything else needs to be addressed!